index.html

```diff
@@ -1,67 +1,218 @@
-                    <a href="#">
-                        <img src="image/linkedin.png" alt="LinkedIn" class="im-age">
-                    </a>
-                </li>
-                <li class="social-li">
-                    <a href="#">
-                        <img src="image/thumbs.png" alt="Thumbs" class="im-age">
-                    </a>
-                </li>
-                <li class="social-li">
-                    <a href="#">
-                        <img src="image/twitter.png" alt="Twitter" class="im-age">
-                    </a>
-                </li>
-                <li class="social-li">
-                    <a href="#">
-                        <img src="image/M.png" alt="M" class="im-age">
-                    </a>
-                </li>
-            </ul>
+    <main class="main">
+        <div class="recent">My Recent Works</div>
+        <div class="grid">
+            <div class="workswidth">
+                <div class="works1">
+                    <div class="yogalady"> <img src="image/yogalady.png" alt="Yogalady"></div>
+                    <div class="multi-post1">Multi-Post Stories</div>
+                    <div class="para-works1">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</div>
+                    <div class="list1">
+                        <ul class="boxes1">
+                            <li class="box box1"><img src="image/css1.png" alt="CSS"></li>
+                            <li class="box box2"><img src="image/html1.png" alt="HTML"></li>
+                            <li class="box box3"><img src="image/bootstrap1.png" alt="Bootstrap"></a></li>
+                            <li class="box box4"><img src="image/ruby1.png" alt="Ruby"></li>
+                        </ul>
+                    </div>
+                    <div><button class="button1">See Project</button></div>
+                </div>
+    
+                <div class="works2">
+                    <div class="image-back">
+                        <div class="professional">Profesional Art Printing Data</div>
+                        <div class="paragraphworks">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</div>
+                        <div>
+                            <ul class="boxes2">
+                                <li class="waks1"><a href="#"></a><img src="image/html2.png" alt="HTML"></li>
+                                <li class="waks2"><a href="#"></a><img src="image/bootstrap2.png" alt="Bootstrap"></li>
+                                <li class="waks3"><a href="#"></a><img src="image/rubby2.png" alt="Ruby"></li>
+                            </ul>
+                        </div>
+                    </div>
+                    <button class="button2">See Project</button>
+                </div>
```

Comment:
- [ ] Great job using the `flexbox` display for the first card of your works section, but as per [the project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md), it's important to use `flexbox` for all of your cards. This will help ensure that the layout is consistent and visually appealing across all cards. 🤓

  ![image](https://user-images.githubusercontent.com/107952724/235326139-a52c874a-35f9-4783-94ca-936b7feb139c.png)

- [ ] **As I suggested above**, you are using a `div` tag with text for your title and for your paragraph. As per `HTML` best practices, would you be able to use an hx tag **(e.g. h1, h2, etc.)** and a p tag for your paragraph instead? This will help ensure that your `HTML` is semantically correct and easier to read for both machines and humans. 💯

Reply:
Okay, I sure will. Kindly enable my 'Request Review' button to 'Changes Required' in my dashboard to enable me to submit again.